script/Arcueid/Day 8/08_00_ARC08_2_1.txt

```diff
@@ -46,7 +46,7 @@ So, unfortunately, I don't think I can help."
 -- Page 119, Offset 19965.
 -- 「なに、長男なのに知らないの？　そりゃクマった。やっぱり妹サンの方が実権にぎってるワケ？」
 "Really? You're the eldest son and you don't know?
- Well that sucks.
+ Well, that sucks.
```

Comment:
I agree. If this was it's own line, I'd probably like the pause since it gives it a sorta shrug-yness to it but I think that in this context I prefer without.